types/fs-extra/fs-extra-tests.ts

```diff
@@ -15,6 +15,9 @@ const modeNum = 0;
 const modeStr = "";
 const object = {};
 const errorCallback = (err: Error) => { };
+const ensureOptions: fs.EnsureOptions = {
```

Comment:
What about tests for providing a `number` as the second parameter?

Reply:
I have added the test for number.